LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2012-present Björn Grüning and other contributors
```

Comment:
@bgruening you think this line is ok?

Reply:
No idea, is that needed? I mean my name and the authors?

Reply:
No idea either, sorry :/

I just copied from IUC. It seems it is in https://mit-license.org/ and BioConda has that line as well.

Reply:
Then lets write:

> Copyright (c) 2012-present The 'galaxytoools' contributors

Reply:
Thanks